src/v/raft/prevote_stm.cc

```diff
@@ -164,7 +164,15 @@ ss::future<bool> prevote_stm::do_prevote() {
     _config->for_each_voter([this](vnode id) { (void)dispatch_prevote(id); });
 
     // process results
-    return process_replies().then([this]() { return _success; });
+    return process_replies().then([this]() {
+        if (_success && _ptr->_node_priority_override == zero_voter_priority) {
+            vlog(
+              _ctxlog.debug,
+              "Ignoring successful pre-vote racing with node drain");
```

Comment:
maybe some more generic log message would be more appropriate ? i mean currently the only thing that sets voter priority is drain manager but in genearal it can be anything, raft does not depend on `cluster` submodule so mentioning node drain in the message is missleading.

Reply:
Good point. I've fixed this to only print information related to priority.

Reply:
https://github.com/redpanda-data/redpanda/compare/4dcbcd829b60322c2b4ba3d785011825fc996a78..b96522ea617be9d1c4d9cdbc4237a59557bd9eb7